src/resources/min-lib.d.ts

```diff
@@ -0,0 +1,19 @@
+// Typescript compiler requires the types below to be defined.
```

Comment:
why is this needed? Clutz always emits within a o_o.clutz namespace, so there should be no collision with `lib.d.ts` and your output.

I want to keep the invariant that clutz' output has to always be consumed together with 'closure.lid.d.ts' and 'lib.d.ts'. Otherwise, we will get into inconsistencies in the output. For example currently we output `clutz.PrivateType` alias for private types, and that is defined in 'closure.lib.d.ts'.